### PR TITLE
Updating meeting details

### PR DESCRIPTION
**Pull request by paulczar**
I was struggling to find the right meeting, I eventually found it at the big [kubernetes calendar](https://calendar.google.com/calendar/u/0/embed?src=calendar@kubernetes.io) after Matt Broberg gave me the time for it.  This update should make it easier to find. 

I believe the meetings link should have been to the `#contributor-comms anchor`, so I updated that too, let me know if I'm wrong.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

File: communication/marketing-team/README.md [hunk 1]
```diff
@@ -37,11 +37,15 @@ the broader ecosystem.
 
 We still have many roles available!  
 
-Come to one of our [meetings] (join kubernetes-sig-contribex@googlegroups.com for the invite)
-or reach out to us on the #sig-contribex channel on the Kubernetes slack (slack.k8s.io for invite).
+* Come to one of our [meetings] (Every Friday at 8am PT), see the [calendar] to add it to your own calendar.
+* Join our [mailing-list],
+* Reach out to us on the #sig-contribex channel on the Kubernetes slack (slack.k8s.io for invite).
+
```
Comment (mbbroberg) [inline communication/marketing-team/README.md:43]:
```suggestion
* Come to one of our [meetings] (Every Friday at 8am PT). See the [calendar] to add it to your own calendar.
* Join our [mailing-list], and/or reach out to us on the #sig-contribex channel on the Kubernetes slack ([slack.k8s.io](http://slack.k8s.io)).

```

**Comment by mbbroberg** [comment]
Thanks for jumping in @paulczar. The tricky bit here is that that information is codified in YAML elsewhere, and it will become out of sync if copied directly into this page. That said, the drift is worth it in my book since it's currently inconveniencing people getting involved. That takes priority over automation and I'll open a bug to deal with the drift. 

TL;DR
/lgtm

**Comment by mbbroberg** [comment]
@paulczar the `@` to me in the comment is why we still have the `do-not-merge/invalid-commit-message` showing up.

**Comment by mbbroberg** [comment]
/approve 

🤦
